run-platform.sh

```
@@ -195,10 +195,9 @@ setup_env() {
   DEFAULT_AUTH_KEY="unstract"
 
   for service in "${services[@]}"; do
-    # Skip services that are spawned at runtime
-    for ignore_service in "${spawned_services[@]}"; do
+    for ignore_service in "${ignore_services[@]}"; do
```

Shellcheck (0.11.0)

[warning] ignore_services is referenced but not assigned (did you mean 'ignore_service'?).

(SC2154)